endpoint.rst

```diff
@@ -85,29 +39,25 @@ You can interact with the public Sylabs Cloud using various
 `instance
 <cli/{command}_instance.html>`_
 
-.. note::
-
-   Using ``docker://``, ``oras://`` and ``shub://`` URIs with these
-   commands does not interact with the Sylabs Cloud.
-
 ***************************
  Managing Remote Endpoints
 ***************************
 
+A fresh installation of {Project} is not configured to connect to a Library
+API endpoint and one should be set by the system administrator or users.
+Well known endpoint options are listed in the `Library API Registry Docs
+<https://singularityhub.github.io/library-api/#/?id=library-api>`_.
+
```

Comment:
Please add something that explicitly says that the remote called "DefaultRemote" does not support the Library API.  It only has a key server, which is the openpgp.org public key server.